### ISSUE TITLE
Enable Stats view for non-admins

### ISSUE THREAD

**Issue by miskolc** [chakki-works/doccano#270]
My company will be using Doccano for text classification and we want to show the annotation user stats page to some one the non-admin users.

Can you provide this permission so that I can added to users.

**Comment by Hironsan** [comment]
Hello @miskolc.

One quick solution is to remove `IsAdminUserAndWriteOnly` permission from `StatisticsAPI`:

* https://github.com/chakki-works/doccano/blob/50c7078c16e69575d069f110b7a39925b5f732c3/app/api/views.py#L65

But I think it's not a simple problem to remove the permission. If non-admin users can see annotation statistics, their decision can be affected by it. For example, if the stats show that the 90% labels are positive and 10% are negative, their decision will be biased.

I think it's ok that non-admin users can see stats about their own annotation, but if they can see other users' stats, I'm against it.

**Comment by rbagd** [comment]
I agree that seeing labels annotations of other users might influence the decision process and should not happen by default. In addition to @Hironsan's suggestion, a quick solution is to comment out these lines from the stats view, though it will affect everyone, admin and non-admin.

https://github.com/chakki-works/doccano/blob/50c7078c16e69575d069f110b7a39925b5f732c3/app/server/static/components/stats.vue#L14-L15

Of course, hiding the plot does not hide the data which still can be retrieved via the API.

A nice add-on for the stats view would be possibility to grant permissions for the user see only overall number of documents that have been annotated across the project by user. It's handy as user can compare his/her own progress with respect to other users . Of course, this assumes that annotated document != some particular label.

**Comment by c-w** [comment]
It may be worthwhile to move statistics out of doccano and into a separate app (e.g. Grafana as mentioned in https://github.com/chakki-works/doccano/issues/114#issuecomment-475770975). This would provide complete freedom for the types of visualizations and dashboards that people can build without having to modify doccano's API to fit all use cases (which may be very difficult to do).
